config/v1/types_operatorhub.go

```diff
@@ -38,30 +46,24 @@ type OperatorHubList struct {
 	Items           []OperatorHub `json:"items"`
 }
 
-// HubSource is used to specify the OperatorSource and its configuration
+// HubSource is used to specify the hub source and its configuration
 type HubSource struct {
-	// name is the name of one of the default OperatorSources
+	// name is the name of one of the default hub sources
 	// +kubebuilder:validation:MaxLength=253
 	// +kubebuilder:validation:MinLength=1
 	// +kubebuilder:Required
 	Name string `json:"name"`
-	// disabled is used to disable a default OperatorSource on cluster
+	// disabled is used to disable a default hub source on cluster
 	// +kubebuilder:Required
 	Disabled bool `json:"disabled"`
 }
 
 // HubSourceStatus is used to reflect the current state of applying the
 // configuration to a default source
 type HubSourceStatus struct {
-	// name is the name of one of the default OperatorSources
-	// +kubebuilder:validation:MaxLength=253
-	// +kubebuilder:validation:MinLength=1
-	// +kubebuilder:Required
-	Name string `json:"name,omitempty"`
-	// configuration is the state of the default OperatorSources configuration
-	Configuration map[string]string `json:"configuration,omitempty"`
+	HubSource
```

Comment:
Not having a json tag is an issue when generating `openAPIValidation` block in CCO. I will fix this and whatever other changes are requested in a follow up PR.